web/app.php

```diff
@@ -8,11 +8,10 @@
 // Use APC for autoloading to improve performance.
 // Change 'sf2' to a unique prefix in order to prevent cache key conflicts
 // with other applications also using APC.
-/*
-$apcLoader = new ApcClassLoader('sf2', $loader);
+$apcLoader = new ApcClassLoader('ilios', $loader);
```

Comment:
We might need to be more specific here since we (for example) have 2-3 instances of ilios-stage running on the same machine.    Symfony is using a hash of the file path now - that seems like a good solution https://github.com/symfony/symfony-standard/blob/master/web/app.php#L16